### PR TITLE
feat: improve pre-requisites

### PR DESCRIPTION
**Pull request by nchabanoles**
Add notes to clarify pre-requisites.

Explicitly mention the supported versions of BPA and what to do if the uses has an older version of Bonita. Also, explain that other DB vendors are available upon request.

### CODE REVIEW

**Comment by uguy** [comment]
@Yves-Jacq no technical point on this one. Can you do the review ?

File: modules/cli/pages/index.adoc [hunk 1]
```diff
@@ -22,12 +22,14 @@ You can retrieve a license by logging into your Bonita Process Insights instance
...
 == Export data
+=== Pre-requisites
+The tool is currently optimized to export data from version 2021.2 (7.13.0) and beyond. While it may be working for older versions, you should contact Bonitasoft to ensure it.
 
 === Configuration
 
 Before exporting, you need to configure the connection to the Bonita Instance from which you want to export the data.
 
-Export tool can support two types of databases PostgreSQL and Oracle. The JDBC URL must be adapted according to the type of database you will use.
+Export tool can support two types of databases PostgreSQL and Oracle (SQLServer and MySQL are also available upon demand). The JDBC URL must be adapted according to the type of database you will use.
```
Comment (tbouffard) [inline modules/cli/pages/index.adoc:32]: **nitpick:** This must be validated by @Yves-Jacq

**Review by Yves-Jacq — APPROVED** [review]
Ok for me

**Review by tbouffard — APPROVED** [review]
Approved by stakeholders, so merging

**Pull request merged**